docs/sources/developers/plugins/create-a-grafana-plugin/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+description: How-to topics for plugin development
+title: Create a Grafana plugin
+---
+
+This section contains how-to topics for developing Grafana plugins.
+
+- [Build a Grafana plugin](https://grafana.github.io/plugin-tools/docs/creating-a-plugin)
+    - [Build a panel plugin](https://grafana.com/tutorials/build-a-panel-plugin/)
+    - [Build a data source plugin](https://grafana.com/tutorials/build-a-data-source-plugin/)
+    - [Build a data source backend plugin](https://grafana.com/tutorials/build-a-data-source-backend-plugin/)
+    - [Build a logs data source plugin](build-a-logs-data-source-plugin.md)
+    - [Build a streaming data source plugin](build-a-streaming-data-source-plugin.md)
+- [Customize a Grafana plugin
```

Comment:
Open brackets / missing link. Still not a fan of customize as the verb

Reply:
How about we go with "extend" after all. This doesn't have a page to link to at this time, so no broken link. We could create one later.